### PR TITLE
created the index.html file

### PR DESCRIPTION
**Pull request by hundrow**
I wrote the index page for a website that will use ajax to fill out the first part of the page from a file called header.html and the bottom part of the page from a file called footer.html.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    <h1>This is the h1 inside the body</h1>
+    <p>a paragraph inside of the body</p>
+</div>
+<div>
+    <button id="button1" onclick="displayMessage(1)">button1</button>
+    <button id="button2" onclick="displayMessage(2)">button1</button>
+    <p id="message"></p>
+</div>
+    <script>
+        function displayMessage(buttonNumber) {
```
Comment (jaxrosco) [inline index.html:15]: Can you explain this function?

Comment (hundrow) [reply index.html:15]: the function changes the message text based on which button is pressed by the user.

Comment (jaxrosco) [reply index.html:15]: Thank you!

**Review by jaxrosco — APPROVED** [review]
Looks good!

**Pull request merged**